schema/profile-to-establishment.js

```diff
@@ -0,0 +1,41 @@
+const BaseModel = require('./base-model');
+
+class ProfileToEstablishment extends BaseModel {
+  static get tableName() {
+    return 'profile_to_establishment';
+  }
+
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      required: ['establishmentId', 'profileId'],
+      additionalProperties: false,
+      properties: {
+        establishmentId: { type: 'integer' },
+        profileId: { type: 'string' },
+        createdAt: { type: 'string', format: 'date-time' },
+        updatedAt: { type: 'string', format: 'date-time' },
+        deleted: { type: ['string', 'null'] }
+      }
+    };
+  }
+
+  static get relationMappings() {
+    return {
+      profile: {
+        relation: this.BelongsToOneRelation,
+        modelClass: `${__dirname}/profile`,
+        join: {
+          from: 'profile_to_establishment.profileId',
+          to: 'profiles.id'
+        }
+      }
+    };
+  }
+
+  getProfile() {
+    return this.$relatedQuery('profile');
+  }
```

Comment:
we don't need this, this model only needs to define the jsonSchema

Reply:
done